odl/contrib/datasets/ct/examples/mayo_reconstruct.py

```diff
@@ -0,0 +1,48 @@
+"""Reconstruct Mayo dataset using FBP and compare to reference recon.
+
+Note that this example requires that Mayo has been previously downloaded and is
+stored in "mayo_folder".
+
+In this example we only use a subset of the data for performance reasons,
+there are ~48 000 projections in the full dataset.
+"""
+
+import odl
+from odl.contrib.datasets.ct.mayo import (projections_from_folder,
+                                          volume_from_folder)
+
+mayo_folder = 'E:/Data/MayoClinic'  # replace with your local folder
+
+# Load reference reconstruction
+volume_folder = mayo_folder + '/Training Cases/L067/full_1mm_sharp'
+partition, volume = volume_from_folder(volume_folder)
```

Comment:
Weird to have a function called `volume_from_...` return something else *and* the volume.

For functions that return multiple things it's actually okay IMO to use a verb, here for example `load_reference`.